vpr/src/draw/draw.cpp

```diff
@@ -3767,12 +3772,21 @@ static void highlight_blocks(double x, double y) {
 }
 void set_net_alpha_value(GtkWidget* widget, gint /*response_id*/, gpointer /*data*/) {
     std::string fa(gtk_entry_get_text((GtkEntry*)widget));
-    net_alpha = std::stof(fa);
+    t_draw_state* draw_state = get_draw_state_vars();
+    draw_state->net_alpha = std::stof(fa);
+    application.refresh_drawing();
+}
+
+void set_net_alpha_value_with_enter(GtkWidget* widget, gint /*response_id*/, gpointer /*data*/) {
+    std::string fa(gtk_entry_get_text((GtkEntry*)widget));
+    t_draw_state* draw_state = get_draw_state_vars();
+    draw_state->net_alpha = std::stof(fa);
     application.refresh_drawing();
 }
```

Comment:
This looks identical to `set_net_alpha_value`. Could just use the same function rather than duplicating the code?